### DISCUSSION TITLE
Sniffing Interfaces

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#8418] [General] [answered]
Hi,
new to security onion and i want to start using it. i need to clarify things about network sniffing:
1. can i label each sniffing card ?
2. can i use each sniffing card individually and not in a bond interface ?
3. if i have to use bond interface, can i filter information according to the interface the sniffing occurred on ?

Regards,
Udi

**Comment by reyesj2** [comment, marked as answer]
Currently not possible all data is ingested through a bonded interface of however many monitor ports you configure at setup. Can I ask what you're looking to accomplish by filtering traffic by individual monitor port?

**Reply by udi-mosh** [reply]
i am sniffing the network in different points of the network and i came to the conclusion that having the ability to know from which part of the network i got the packet is crucial and helpful. i want to be able to create filters and dashboards not only based on source and destination.

**Reply by udi-mosh** [reply]
also, it helps when we have LAG interfaces

**Reply by udi-mosh** [reply]
i will try to give you an example.
let's say i have server segment and users segment and i configure firewall rules to allow certain protocols between them.
if i place a TAP in the uplink of each segment i have the ability upon searching for anomalies, to see if packets went through even though they were not suppose to according to firewall rules.
can i reach that level using the bond interface ? can i see if packet went from users to servers got through even though it shouldn't ?

**Reply by udi-mosh** [reply]
come to think about, is it possible to create several bond interfaces ?

**Reply by dougburks** [reply]
> i will try to give you an example.
let's say i have server segment and users segment and i configure firewall rules to allow certain protocols between them.
if i place a TAP in the uplink of each segment i have the ability upon searching for anomalies, to see if packets went through even though they were not suppose to according to firewall rules.
can i reach that level using the bond interface ? can i see if packet went from users to servers got through even though it shouldn't ?

If you have a sniffing interface in the server segment and a sniffing interface in the users segment, then the bond will include traffic from both segments. Security Onion should then generate metadata logs based on all traffic seen in both segments. You should then be able to search those metadata logs to see if there were any connections between those segments that shouldn't have been allowed by the firewall rules.

If you really feel the need to label the segments, then one option would be to build a distributed deployment with one forward node for the server segment and one forward node for the users segment. You could then use the hostname of the forward node to differentiate between the segments and as a filter in your queries.

**Reply by udi-mosh** [reply]
thank you very much for the response

**Comment by TotieBash** [comment]
Similar goal objective to #7747.  I have similar desire for what you are trying to accomplish if that makes you feel better =).

**Reply by udi-mosh** [reply]
Glad to see i am not alone.
